### PR TITLE
Fixed the warnings for deprecated events manager.

### PR DESCRIPTION
**Pull request by vincethecoder**
- Resolved warnings for deprecated `EventsManager`, renamed `NavigationEventsManager`.

/cc @akitchen @JThramer @frederoni

### CODE REVIEW

File: MapboxNavigation/FeedbackViewController.swift [hunk 1]
```diff
@@ -106,7 +106,7 @@ public class FeedbackViewController: UIViewController, DismissDraggable, UIGestu
     }
     
     /**
-     Initialize a new FeedbackViewController from an `EventsManager`.
+     Initialize a new FeedbackViewController from an `NavigationEventsManager`.
```
Comment (akitchen) [inline MapboxNavigation/FeedbackViewController.swift]: Very minor grammar nit: "an" -> "a"

File: MapboxNavigation/NavigationViewController.swift [hunk 2]
```diff
@@ -742,7 +742,7 @@ public extension NavigationViewController {
         }
     }
     
-    @available(*, obsoleted: 0.1, renamed: "navigationService.eventsManager", message: "NavigationViewController no-longer directly manages an EventsManager. See MapboxNavigationService, which contains a reference to the eventsManager, for more information.")
+    @available(*, obsoleted: 0.1, renamed: "navigationService.eventsManager", message: "NavigationViewController no-longer directly manages an NavigationEventsManager. See MapboxNavigationService, which contains a reference to the eventsManager, for more information.")
```
Comment (akitchen) [inline MapboxNavigation/NavigationViewController.swift]: "an" -> "a"

File: MapboxNavigation/FeedbackViewController.swift (hunk 1 above)
Comment (akitchen) [reply MapboxNavigation/FeedbackViewController.swift]: Don't forget to fix up [this change](https://github.com/mapbox/mapbox-navigation-ios/pull/1786/files#diff-a25e8b3bfe9ceec368b83a004a6d1708R118) below

**Comment by JThramer** [comment]
Looking better, but this shouldn't be merged until [the encoder bug](https://github.com/mapbox/mapbox-navigation-ios/pull/1786/files#diff-a25e8b3bfe9ceec368b83a004a6d1708R118) is fixed. Just wanted to make sure this doesn't fall-through the cracks.

**Pull request merged**